sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/OpenTelemetryBuilderExtensions.cs

```diff
@@ -21,24 +21,52 @@ namespace Azure.Monitor.OpenTelemetry.AspNetCore
     public static class OpenTelemetryBuilderExtensions
     {
         /// <summary>
-        /// Adds Azure Monitor Defaults into <see cref="OpenTelemetryBuilder" />.
+        /// Configures Azure Monitor for logging, distributed tracing, and metrics.
         /// </summary>
         /// <param name="builder"><see cref="OpenTelemetryBuilder"/>.</param>
         /// <returns>The supplied <see cref="OpenTelemetryBuilder"/> for chaining calls.</returns>
-        public static OpenTelemetryBuilder WithAzureMonitor(this OpenTelemetryBuilder builder)
+        /// <remarks>
+        /// This method configures Azure Monitor for use with OpenTelemetry by adding the Azure Monitor exporter for logging,
+        /// distributed tracing, and metrics. It also configures the OpenTelemetry logger to include formatted messages and
+        /// parsed state values.
+        ///
+        /// The following instrumentation is added for distributed tracing:
```

Comment:
```suggestion
        /// The following instrumentations are added for distributed tracing:
```

Reply:
Having list type would be better here?

Reply:
No strong preference, I could give it a try.

Reply:
Looks better now with para and list.

![image](https://user-images.githubusercontent.com/9479006/230276625-5fa7d37e-a8d2-4dcf-97fd-4df7f3b65750.png)